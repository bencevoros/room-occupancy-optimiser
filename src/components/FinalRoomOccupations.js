function FinalRoomOccupations({ roomOccupations }) {
  return (
    <div data-testid="finalRoomOccupations">
      <p><b>Usage Economy:</b> {roomOccupations.usageEconomy.rooms} (EUR {roomOccupations.usageEconomy.price})</p>
      <p><b>Usage Premium:</b> {roomOccupations.usagePremium.rooms} (EUR {roomOccupations.usagePremium.price})</p>
    </div>
  );
}

export default FinalRoomOccupations;
