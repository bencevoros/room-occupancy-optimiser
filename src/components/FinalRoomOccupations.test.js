import { render, screen } from '@testing-library/react';
import FinalRoomOccupations from './FinalRoomOccupations';

const roomOccupations = {
  usageEconomy: { rooms: 0, price: 0 },
  usagePremium: { rooms: 0, price: 0 },
}

describe('FinalRoomOccupations', () => {
  describe('calls setRoomOccupations', () => {
    test('when component renders', () => {
      render(<FinalRoomOccupations roomOccupations={roomOccupations} />);
      expect(screen.queryByTestId('finalRoomOccupations')).toMatchSnapshot()
    });
  });
});
