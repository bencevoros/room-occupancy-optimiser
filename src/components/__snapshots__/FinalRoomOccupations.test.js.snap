// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FinalRoomOccupations calls setRoomOccupations when component renders 1`] = `
<div
  data-testid="finalRoomOccupations"
>
  <p>
    <b>
      Usage Economy:
    </b>
     
    0
     (EUR 
    0
    )
  </p>
  <p>
    <b>
      Usage Premium:
    </b>
     
    0
     (EUR 
    0
    )
  </p>
</div>
`;
